backend/src/main/scala/cromwell/backend/standard/callcaching/StandardCacheHitCopyingActor.scala

```diff
@@ -178,8 +178,12 @@ abstract class StandardCacheHitCopyingActor(val standardParams: StandardCacheHit
           stay() using Option(newData)
       }
     case Event(IoFailure(command: IoCommand[_], failure), Some(data)) =>
-      // any failure is fatal
-      context.parent ! JobFailedNonRetryableResponse(jobDescriptor.key, failure, None)
+      // We can get extra I/O timeout messages from previous cache copy attempt. This is due to the fact that if one file fails to copy,
+      // we immediately cache miss and try the next potential hit, but don't cancel previous copy requests.
+      // (any other failure is fatal)
+      if (!failure.isInstanceOf[TimeoutException]) {
```

Comment:
Why the check on `TimeoutException` ? If it is a timeout then aren't we going to get the EJEA stuck waiting for a response ?

Reply:
this is the message that the EJEA was dropping on the floor previously

Reply:
Per discussion in chat:

Since this actor is only responsible for copying a single cache hit, I don't think it should be able to say that all cache-hit copying attempts should cease and the job should run.

Reply:
I think we're just re-using `JobFailedNonRetryableResponse` and changing it's meaning when we really want something like a `JobCacheHitCopyingFailed`